universal-application-tool-0.0.1/app/auth/ProfileFactory.java

```diff
@@ -38,4 +40,12 @@ private UatProfileData create(Roles role) {
     p.addRole(role.toString());
     return p;
   }
+
+  public UatProfile wrapAccount(Account account) {
+    return wrapProfileData(new UatProfileData(clock, account.id));
+  }
+
+  public UatProfile wrapApplicant(Applicant applicant) {
```

Comment:
(not blocking): i'm not sure if it would be clearer or more confusing, but you could overload `wrap(Account account)` and `wrap(Applicant applicant)`, in case you hadn't considered it.